engine/include/cubos/engine/input/axis.hpp

```diff
@@ -74,11 +75,20 @@ namespace cubos::engine
         /// @param value New value.
         void value(float value);
 
+        /// @brief Gets the deadzone.
+        /// @return Deadzone.
+        float deadzone() const;
+
+        /// @brief Sets the deadzone.
+        /// @param deadzone New deadzone.
+        void setDeadzone(float deadzone);
```

Comment:
```suggestion
        void deadzone(float deadzone);
```
We usually omit the `set` and `get` prefixes, for consistency with other parts of the code.